docs/modules/ROOT/pages/context-population.adoc

```diff
@@ -0,0 +1,50 @@
+= Context population
+
+The request context can be populated with a number of different values to influence the behaviour of the Neo4j GraphQL Library.
+
+== `driverConfig`
+
+== `executionContext`
+
+The Neo4j driver, session or transaction which will be used to execute the translated query.
+
+== `jwt`
+
+A decoded JWT payload A decoded JWT payload which can be provided for use in authentication and authorization.
```

Comment:
This would be another case where having an index page for the Authentication and Authorization would be useful.

```suggestion
A decoded JWT payload which can be provided for use in Authentication and Authorization.
```